src/Raven.Server/Documents/Patch/DocumentPatcherBase.cs

```diff
@@ -129,6 +129,10 @@ private void PrepareEngine(PatchRequest patch, PatcherOperationScope scope, Engi
 
             CustomizeEngine(jintEngine, scope);
 
+            jintEngine.SetValue("$now", JsValue.FromObject(jintEngine, DateTime.Now));
```

Comment:
It would be better to make these into functions, not static values.
A major issue here is that we are caching the engine, so that can't be made to work with statically cached values.